src/main/kotlin/ink/abb/pogo/scraper/tasks/ProcessPokestops.kt

```diff
@@ -42,13 +43,22 @@ class ProcessPokestops(var pokestops: MutableCollection<Pokestop>) : Task {
             val distanceB = self.getEarthDistance(locationB)
             distanceA.compareTo(distanceB)
         })
-        if (startPokeStop == null)
-            startPokeStop = sortedPokestops.first()
+        if (startPokestop == null)
+            startPokestop = sortedPokestops.first()
 
         if (settings.lootPokestop) {
             val loot = LootOneNearbyPokestop(sortedPokestops, lootTimeouts)
             bot.task(loot)
         }
+        if (settings.campLurePokestop > 0) {
+            val luresInRange = sortedPokestops.filter {
+                it.inRangeForLuredPokemon() && it.fortData.hasLureInfo()
+            }.size
+            if (luresInRange >= settings.campLurePokestop) {
```

Comment:
@jabbink shouldn't you check if catchPokemon in settings is true? If it's not there isn't any way to catch the lured pokemons.